src/components/cc-orga-member-card/cc-orga-member-card.stories.js

```diff
@@ -0,0 +1,242 @@
+import './cc-orga-member-card.js';
+import { makeStory, storyWait } from '../../stories/lib/make-story.js';
+import { enhanceStoriesNames } from '../../stories/lib/story-names.js';
+
+const baseMember = {
+  state: 'loaded',
+  id: 'user_c78e5148-a8c2-4c22-b180-2b88ec531f0a',
+  email: 'john.doe@example.com',
+  role: 'DEVELOPER',
+  newRole: 'DEVELOPER',
+  name: 'John Doe',
+  avatar: 'http://placekitten.com/200/200',
+  jobTitle: 'Frontend Developer',
+  isMfaEnabled: false,
+  isCurrentUser: false,
+};
+
+const longEmail = 'very-very-very-very-very-very-very-very-very-very-very-very-very-very-very-very-very-very-very-very-very-very-long-email-address@very-very-very-very-very-very-very-long-example.com';
```

Comment:
Tiny detail but please use a subdomain of `example.com`.

=> `very-very-very-very-very-very-very-very-very-very-very-very-very-very-very-very-very-very-very-very-very-very-long-email-address@very-very-very-very-very-very-very-long.example.com`